benchmark_runner/common/remote_ssh/ssh_remote_exceptions.py

```diff
@@ -0,0 +1,67 @@
+class SshError(Exception):
+    """ Base class for all SSH error classes.
+        All exceptions raised by the DSS library should inherit from this class. """
+    pass
+
+
+class RunCommandError(SshError):
+    """
+    This class for run command error
+    """
+    def __init__(self, err):
+        self.message = f'Cannot run shell command: {err}'
+        super(RunCommandError, self).__init__(self.message)
+
+
+class SshConnectionError(SshError):
+    """
+    This class for connection error
+    """
+    def __init__(self):
+        self.message = f'Missing connection credentials'
```

Comment:
Might want to include the remote host and command to assist in debugging (presumably the connection can fail for a variety of reasons).

Reply:
Still open.